taxonomies/categories.txt

```diff
@@ -48379,6 +48379,7 @@ nl:Kokosboter, Kokosboters
 
 #do not put in nut butters, peanut is not a nut
 <en:Legume butters
+<en:Nut butters
```

Comment:
What about the comment just above ?

Reply:
Well peanut is not a nut. For allergens they differentiate between peanuts and tree nuts (hazel, wal, ...)

Reply:
The distinction is also important for the calculation of the nutri-score I think

Reply:
Indeed, I didn't see the comment. Many contributors make this mistake though, so it's quite frequent that "Nut butters" is suggested as a category for peanut butters.